app/controllers/trajectories_controller.rb

```diff
@@ -0,0 +1,4 @@
+class TrajectoriesController < ApplicationController
+  def index 
+    Trajectory.where(city_id: params.fetch(:city_id,[]))
```

Comment:
🤔 how does your view handle this? and why a fetch? 
what if there is no trajectory?

Reply:
help @mtopete1996  help

Reply:
```suggestion
    Trajectory.where(city_id: params[:city_id])
```

This would work the same way